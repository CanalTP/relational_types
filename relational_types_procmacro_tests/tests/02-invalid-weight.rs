mod test_utils;

use relational_types::*;
use test_utils::*;
use typed_index_collection::*;

#[derive(GetCorresponding)]
pub struct Model {
    #[get_corresponding(weight = "abc")]
    animals_to_felines: OneToMany<Animal, Feline>,
}

fn main() {}
