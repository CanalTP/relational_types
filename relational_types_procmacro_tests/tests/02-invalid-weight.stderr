error: proc-macro derive panicked
 --> $DIR/02-invalid-weight.rs:6:10
  |
6 | #[derive(GetCorresponding)]
  |          ^^^^^^^^^^^^^^^^
  |
  = help: message: `weight` attribute must be convertible to f64: ParseFloatError { kind: Invalid }
